content/docs/studio/get-started.md

```diff
@@ -9,18 +9,20 @@ https://www.youtube.com/watch?v=hKf4twg832g
    using your GitHub.com, GitLab.com or Bitbucket.org account, or your email
    address.
 
-<admon type="info">
+   <admon>
 
-When you sign up, you're on the **Free plan** by default. To sign up for the
-**Team plan**, you should first [create a team](/doc/studio/user-guide/teams).
-Then go to the [team's settings page](/doc/studio/user-guide/teams#settings) and
-in the `Plan and billing` section, click on `Change plan`. To sign up for the
-**Enterprise plan**,
-[contact us](https://calendly.com/gtm-2/studio-introduction). To compare the
-different plans, check out our
-[pricing details](https://studio.iterative.ai/pricing).
+   When you sign up, you're on the **Free plan** by default. To switch to the
+   **Team plan**, [create a team] first, then go to the [team's settings] and
+   click on `Change plan` in the `Plan and billing` section. switch to the
+   **Enterprise plan**, please [contact us].  
+   See [pricing details] to compare the different plans.
 
-</admon>
+   [create a team]: /doc/studio/user-guide/teams
+   [team's settings]: /doc/studio/user-guide/teams#settings
+   [contact us]: https://calendly.com/gtm-2/studio-introduction
```

Comment:
There's at least 3 different links to `contact us`. One of them is this calendly thing, another one is just a `mailto:support@iterative.ai`, and finally there's https://dvc.org/doc/studio/troubleshooting#support which I think should be the only one throughout... ⌛